src/daft-sql/src/planner.rs

```diff
@@ -1900,25 +1916,24 @@ impl<'a> SQLPlanner<'a> {
                 }
                 None => DataType::Float64,
             },
-            SQLDataType::Float4 | SQLDataType::Real => DataType::Float32,
+            SQLDataType::Real | SQLDataType::Float32 => DataType::Float32,
 
             // ---------------------------------
             // decimal
             // ---------------------------------
-            SQLDataType::Dec(info) | SQLDataType::Decimal(info) | SQLDataType::Numeric(info) => {
-                match *info {
-                    ExactNumberInfo::PrecisionAndScale(p, s) => {
-                        DataType::Decimal128(p as usize, s as usize)
-                    }
-                    ExactNumberInfo::Precision(p) => DataType::Decimal128(p as usize, 0),
-                    ExactNumberInfo::None => DataType::Decimal128(38, 9),
+            SQLDataType::Dec(_) | SQLDataType::Numeric(_) => use_instead!(dtype, "`decimal`"),
```

Comment:
nit: `DEC` is an ANSI-SQL alias for DECIMAL, and most implementations (including postgres) alias NUMERIC (also ANSI-SQL) to DECIMAL even though they are *slightly* different. If we want to follow ANSI-SQL and/or Postgres, then I would leave the dec/numeric aliases in.